.github/actions/build-and-push/action.yaml

```diff
@@ -0,0 +1,54 @@
+name: Build and Push
+description: Build and Push docker image to dibbs-query-connector registry
+
+inputs:
+  username:
+    description: GitHub username of the user who triggered the workflow
+    required: true
+  password:
+    description: Used to log in to the Container registry
+    required: true
+
+runs:
+  using: composite
+  steps:
+    - name: Check Out Changes
+      uses: actions/checkout@v4
+
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v2
+
+    - name: Log in to the Container registry
+      uses: docker/login-action@v2
+      with:
+        registry: ghcr.io
+        username: ${{ inputs.username }}
+        password: ${{ inputs.password }}
+
+    - name: lowercase repo name to use in following step
+      shell: bash
+      env:
+        REPO: ${{ github.repository }}
+      run: |
+        echo "REPO=${GITHUB_REPOSITORY,,}" >> $GITHUB_ENV
+
+    - name: Build and push
+      id: build_push
+      uses: docker/build-push-action@v3
+      with:
+        context: ./query-connector
+        push: true
+        tags: |
+          ghcr.io/${{ env.REPO }}/query-connector:main, ghcr.io/${{ env.REPO }}/query-connector:latest
+
+    # - name: Output image digest
```

Comment:
Is this commented code needed? If not, consider deleting.

Reply:
Done.